faqs/galaxy/galaxy_creating_an_account.md

```diff
@@ -0,0 +1,29 @@
+---
+title: How to create an account on a public Galaxy instance?
+area: Account
+layout: faq
+box_type: tip
+contributors: [jennaj, bernandez]
+---
+
+
+- Having your own account at any public Galaxy instance means that you can save histories and workflows, work with more stored data (up to 250 GB), and get the most out of Galaxy's functionality.
+
+- To create an account at any public Galaxy instance, direct your web browser to [Galaxy Platform](https://galaxyproject.org/use/).
```

Comment:
@bernandez I understood that we would have clear numbered steps summarising this